project/tests/test_delete_request.py

```diff
@@ -0,0 +1,20 @@
+from http import HTTPStatus
+
+from django.test import Client, TestCase
+from django.urls import reverse
+
+from silk.models import Request
+
+
+class TestDeleteRequest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.request = Request()
+        self.request.path = reverse('silk:requests')
+        self.request.method = 'get'
+        self.request.body = b'a' * 1000
+        self.request.save()
+
+    def test_delete(self):
+        response = self.client.post(reverse('silk:request_detail', kwargs={'request_id': str(self.request.id)}))
+        self.assertEqual(response.status_code, HTTPStatus.FOUND)
```

Comment:
Thanks for the tests! A few nitpicks:
- Can you make sure that this re-routes to the summary page and the object is deleted. (Bonus for test that checks the redirection for header Referer)
- Another test that checks when we pass in a non-existing request